boxnav/boxnav/boxnavigator.py

```diff
@@ -4,10 +4,13 @@
 from pathlib import Path
 from random import choice, random, randrange, uniform
 from typing import Callable
+from time import sleep
```

Comment:
Where is sleep being used?

Reply:
It was being used for debugging early on but it's not being used now